### DISCUSSION TITLE
Application not starting due to database connection error

### DISCUSSION THREAD

**Discussion by johan--** [tsirysndr/music-player#178] [Q&A]
When starting the application, it does not find the database connection:

music-player -h

thread 'main' panicked at 'Could not connect to database: Conn("error returned from database: (code: 14) unable to open database file")', /home/runner/work/music-player/music-player/storage/src/lib.rs:26:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**Comment by tsirysndr** [comment]
Hey @johan-- ,

Thanks for reporting this issue,

The auto initialization of the database is not yet implemented for the desktop version, but you can still use the CLI to initialize the configs and the database. https://github.com/tsirysndr/music-player/releases/download/v0.2.0-alpha.14/music-player_v0.2.0-alpha.14_x86_64-unknown-linux-gnu.tar.gz

See https://github.com/tsirysndr/music-player/issues/124
